core/includes/install.core.inc

```diff
@@ -1347,14 +1341,17 @@ function install_select_language(&$install_state) {
     // performing an interactive installation, inform the user that the
     // installer can be translated. Otherwise we assume the user knows what he
     // is doing.
-    if (count($files) == 1) {
+    if (count($standard_languages) == 1) {
```

Comment:
Caution: if connection to server failed, `$standard_languages` will be `FALSE`.
That leads to a TypeError with count().

```
if ($standard_languages && count($standard_languages) == 1)
```
But that's still not enough, as further down it's also assumed this is set and an array...

```

    Warning: foreach() argument must be of type array|object, bool given in install_select_language_form() (line 1423 of /var/www/bdroptesting/html/core/includes/install.core.inc).
    TypeError: count(): Argument #1 ($value) must be of type Countable|array, bool given in install_select_language_form() (line 1440 of /var/www/bdroptesting/html/core/includes/install.core.inc).
```

Reply:
Noted in other comment, `$standard_languages` should always at minimum have English, hence the assumption `count()` being 1

Reply:
Should be fixed